machine-learning-box/sentiment-analysis/py_scripts/data.py

```diff
@@ -6,6 +6,7 @@
 
 os.system(f"{sys.executable} -m pip install -U pytd==0.8.0 td-client")
```

Comment:
Just noticed `import sys` is missing before this line.

Reply:
oh my bad... 
0471d86